src/components/AddFood.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import 'bulma/css/bulma.css';
+
+
+function AddFood(props) {
```

Comment:
this approach of passing the information of a form via the event is perfectly fine. But in react, while working with forms and inputs it is more common to change the values with the state. That way you can pass the information in the state to function as a parameter. See the solution sent to see what I mean. Or check how you did it for the quantity on FoodBox.